xapian-core/tests/api_sorting.cc

```diff
@@ -319,3 +319,49 @@ DEFINE_TESTCASE(sortfunctorremote1, remote) {
     );
     return true;
 }
+
+DEFINE_TESTCASE(replace_weights, backend) {
```

Comment:
Conventionally this would be named `replace_weights1`, especially as there's a `replace_weights2`.